src/scheduling/cost_models/cost_models.h

```diff
@@ -16,5 +16,6 @@
 #include "scheduling/cost_models/sjf_cost_model.h"
 #include "scheduling/cost_models/trivial_cost_model.h"
 #include "scheduling/cost_models/wharemap_cost_model.h"
+#include "scheduling/cost_models/simulated_quincy_cost_model.h"
```

Comment:
In an ideal implementation, the simulated Quincy model would be a wrapper around the real Quincy model that feeds it with simulated data locality information. We should think about re-engineering it that way.